src/components/PopupWithConfirmation.js

```diff
@@ -0,0 +1,22 @@
+import Popup from "./Popup.js";
+
+export default class PopupWithConfirmation extends Popup {
+  constructor(popupSelector, handleFormSubmit) {
```

Comment:
Needs correction: there is no need to provide the submit handler here, you have ` setSubmitAction` method for that